extension/src/popup/components/account/AccountTabs/index.tsx

```diff
@@ -1,38 +1,170 @@
-import React, { useContext } from "react";
+import React, { useEffect, useContext, useMemo, useState } from "react";
 import { useSelector } from "react-redux";
 import classnames from "classnames";
+import { Icon } from "@stellar/design-system";
+import { useTranslation } from "react-i18next";
+import { createPortal } from "react-dom";
+import { Link, useLocation } from "react-router-dom";
+
 import {
   AccountTabsContext,
   TabsList,
 } from "popup/views/Account/contexts/activeTabContext";
 import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
 import { isCustomNetwork } from "@shared/helpers/stellar";
+import { ROUTES } from "popup/constants/routes";
+import { LoadingBackground } from "popup/basics/LoadingBackground";
+
+import { AccountHeaderModal } from "../AccountHeaderModal";
 import "./styles.scss";
 
+const ManageAssetsModalContent = () => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Link to={ROUTES.searchAsset}>
+        <div className="AccountTabs__modal__item">
+          <div className="AccountTabs__modal__item__icon">
+            <Icon.PlusSquare />
+          </div>
+          <div className="AccountTabs__modal__item__title">
+            {t("Add a token")}
+          </div>
+        </div>
+      </Link>
+      <Link to={ROUTES.manageAssets}>
+        <div className="AccountTabs__modal__item">
+          <div className="AccountTabs__modal__item__icon">
+            <Icon.EyeOff />
```

Comment:
is this icon correct or just a placeholder?

Reply:
Good catch - I need to update this!